### PR TITLE
add Kogito Operator CR example to deploy KIEBot

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

**Comment by spolti** [comment]
@radtriste f.y.i

File: src/kiebot-kogito-cr.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      valueFrom:
+        secretKeyRef:
+          key: GITHUB_APP_ID
+          name: kiebot-secret
+    - name: QUARKUS_GITHUB_APP_APP_NAME
+      valueFrom:
+        secretKeyRef:
+          key: GITHUB_APP_NAME
+          name: kiebot-secret
+    - name: QUARKUS_GITHUB_APP_WEBHOOK_PROXY_URL
```
Comment (radtriste) [inline src/kiebot-kogito-cr.yaml]: I am not sure we need that one. 
See https://github.com/quarkiverse/quarkus-github-app/blob/main/runtime/src/main/java/io/quarkiverse/githubapp/runtime/config/GitHubAppRuntimeConfig.java#L57

Comment (spolti) [reply src/kiebot-kogito-cr.yaml]: maybe we can keep, for example purpose only.

Comment (radtriste) [reply src/kiebot-kogito-cr.yaml]: nope, because it breaks the launch of the application ...

Comment (spolti) [reply src/kiebot-kogito-cr.yaml]: No, I meant, keep as it is, for deployment you edit it as needed, wdyt?

Comment (radtriste) [reply src/kiebot-kogito-cr.yaml]: I don't get what you mean ?

Comment (spolti) [reply src/kiebot-kogito-cr.yaml]: keep this file as example only, but when deploying, you edit it to your needs.

Comment (radtriste) [reply src/kiebot-kogito-cr.yaml]: Well, this file would be used only for prod or ?
for local dev, it is all explained in docs: https://quarkiverse.github.io/quarkiverse-docs/quarkus-github-app/dev/create-github-app.html
I don't see the point of keeping this here

File: kubernetes/kiebot-kogito-cr.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  name: kiebot
+spec:
+  replicas: 1
+  env:
+    - name: QUARKUS_GITHUB_APP_APP_ID
+      valueFrom:
+        secretKeyRef:
+          key: GITHUB_APP_ID
+          name: kiebot-secret
+    - name: QUARKUS_GITHUB_APP_APP_NAME
```
Comment (radtriste) [inline kubernetes/kiebot-kogito-cr.yaml:27]: why did you remove the `QUARKUS_GITHUB_APP_WEBHOOK_SECRET` ?
only the PROXY_URL was problematic ;)

Comment (spolti) [reply kubernetes/kiebot-kogito-cr.yaml:27]: isn't one dependent on each other?

**Comment by radtriste** [comment]
nope there aren't
`QUARKUS_GITHUB_APP_WEBHOOK_SECRET` is for the communication with the webhook
`QUARKUS_GITHUB_APP_PROXY_URL` is for dev purpose, to tell the application to use the smee.io server side events

**Comment by spolti** [comment]
I got, updated, thanks.
Plus added a secret example.

File: kubernetes/secret-example.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+kind: Secret
+metadata:
+  name: kiebot-secret
+  namespace: default
+type: Opaque
+data:
+QUARKUS_GITHUB_APP_APP_ID: 0000001
+QUARKUS_GITHUB_APP_APP_NAME: bot-name
+QUARKUS_GITHUB_APP_WEBHOOK_SECRET: <some_sacret>
+QUARKUS_GITHUB_APP_PRIVATE_KEY: '-----BEGIN RSA PRIVATE KEY----- \
```
Comment (radtriste) [inline kubernetes/secret-example.yaml]:
```suggestion
GITHUB_APP_APP_ID: 0000001
GITHUB_APP_APP_NAME: bot-name
GITHUB_APP_WEBHOOK_SECRET: <some_sacret>
GITHUB_APP_PRIVATE_KEY: '-----BEGIN RSA PRIVATE KEY----- \
```

instead or ?

Comment (spolti) [reply kubernetes/secret-example.yaml]: Ah, yes, correct, thanks :)

File: kubernetes/secret-example.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kiebot-secret
+  namespace: default
+type: Opaque
+data:
+GITHUB_APP_APP_ID: 0000001
+GITHUB_APP_APP_NAME: bot-name
+GITHUB_APP_WEBHOOK_SECRET: <some_sacret>
```
Comment (spolti) [inline kubernetes/secret-example.yaml]:
```suggestion
GITHUB_APP_WEBHOOK_SECRET: <some_secret>
```

**Review by radtriste — APPROVED** [review]
